shell/common/gin_helper/event_emitter_template.cc

```diff
@@ -1,11 +1,14 @@
-// Copyright (c) 2019 Slack Technologies, Inc.
+// Copyright (c) 2023 Salesforce, Inc.
```

Comment:
I'm not sure we're supposed to change existing Copy headers? (tbh idk)

Reply:
i created a whole new file, git decided i renamed it though. not sure what to do. doesn't feel like it matters tho.